test/tools/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+if (DEFINED ENV{OSSFUZZ})
```

Comment:
I think we should add that switch at a higher level and only query a cmake option and not an environment variable here. It would be great if ossfuzz could be visible in the table printed by cmake/EthOptions.cmake.

Reply:
Ack